lib/utils/migration.js

```diff
@@ -0,0 +1,50 @@
+const {TYPE_LINK, TYPE_ARRAY} = require('../contentful')
+/**
+ * Adds utils for the migration
+ *
+ * Example:
+ * const { withUtils } = require('@jungvonmatt/contentful-migrations');
+ *
+ * module.exports = withUtils(async function (migration, context, utils) {
+ *
+ *   ...
+ *
+ *   const ct = await utils.getContentType('contentTypeId');
+ *   const field = await utils.getField('contentTypeId', 'fieldId');
+ *
+ * });
+ *
+ */
+ const getMigrationHelper = (migration, context) => {
+  const { makeRequest } = context;
+
+  return {
+    async getContentType(contentTypeId) {
```

Comment:
Die funktion müsste eher "editOrCreateContentType" heißen.
Ich frage mich, ob man es wirklich unterstützen sollte.
Die Migration sollte doch eigentlich wissen, ob man create oder edit machen muss.
Ich sehe grade keinen Fall, wo es sinnvoll ist in beiden Fällen etwas zu tun.

Reply:
gleiche gedanken hatte ich auch... ich schmeiß es wieder raus :D

Reply:
ist weg